examples/stable_diffusion_v2/train_text_to_image.py

```diff
@@ -62,6 +62,7 @@ def init_env(args):
         device_id=device_id,
         max_device_memory="30GB",  # TODO: why limit?
     )
+    ms.set_context(ascend_config={"precision_mode": "allow_fp32_to_fp16"})    # Only effective on Ascend 901B
```

Comment:
It is the default setting for non-910B devices?

Reply:
other devices do not take effect.